src/dvclive/live.py

```diff
@@ -82,7 +82,7 @@ def __init__(
         resume: bool = False,
         report: Literal["md", "notebook", "html", None] = None,
         save_dvc_exp: bool = True,
-        dvcyaml: Optional[str] = "dvc.yaml",
+        dvcyaml: Optional[str, Path] = "dvc.yaml",
```

Comment:
```suggestion
        dvcyaml: Optional[str, os.Pathlike[str]] = "dvc.yaml",
```

Use `os.PathLike[str]`, which is more generic than `Path`.

Reply:
Hi @skshetry , thanks for reviewing this.
So this PR fixed this [issue](https://github.com/iterative/datachain/issues/386), for `dvcyaml` argument to support `path.Pathlib` value. Will the use of `os.PathLike[str]` be more appropriate here instead of pathlib.Path?